pkg/sdk/config.go

```diff
@@ -87,20 +210,233 @@ func configFile() (string, error) {
+	AccountName                    *string             `toml:"accountname"`
+	OrganizationName               *string             `toml:"organizationname"`
+	User                           *string             `toml:"user"`
+	Username                       *string             `toml:"username"`
+	Password                       *string             `toml:"password"`
+	Host                           *string             `toml:"host"`
+	Warehouse                      *string             `toml:"warehouse"`
+	Role                           *string             `toml:"role"`
+	Params                         *map[string]*string `toml:"params"`
+	ClientIp                       *string             `toml:"clientip"`
+	Protocol                       *string             `toml:"protocol"`
+	Passcode                       *string             `toml:"passcode"`
+	Port                           *int                `toml:"port"`
+	PasscodeInPassword             *bool               `toml:"passcodeinpassword"`
+	OktaUrl                        *string             `toml:"oktaurl"`
+	ClientTimeout                  *int                `toml:"clienttimeout"`
+	JwtClientTimeout               *int                `toml:"jwtclienttimeout"`
+	LoginTimeout                   *int                `toml:"logintimeout"`
+	RequestTimeout                 *int                `toml:"requesttimeout"`
+	JwtExpireTimeout               *int                `toml:"jwtexpiretimeout"`
+	ExternalBrowserTimeout         *int                `toml:"externalbrowsertimeout"`
+	MaxRetryCount                  *int                `toml:"maxretrycount"`
+	Authenticator                  *string             `toml:"authenticator"`
+	InsecureMode                   *bool               `toml:"insecuremode"`
+	OcspFailOpen                   *bool               `toml:"ocspfailopen"`
+	Token                          *string             `toml:"token"`
+	KeepSessionAlive               *bool               `toml:"keepsessionalive"`
+	PrivateKey                     *string             `toml:"privatekey,multiline"`
+	PrivateKeyPassphrase           *string             `toml:"privatekeypassphrase"`
+	DisableTelemetry               *bool               `toml:"disabletelemetry"`
+	ValidateDefaultParameters      *bool               `toml:"validatedefaultparameters"`
+	ClientRequestMfaToken          *bool               `toml:"clientrequestmfatoken"`
+	ClientStoreTemporaryCredential *bool               `toml:"clientstoretemporarycredential"`
+	Tracing                        *string             `toml:"tracing"`
+	TmpDirPath                     *string             `toml:"tmpdirpath"`
+	DisableQueryContextCache       *bool               `toml:"disablequerycontextcache"`
+	IncludeRetryReason             *bool               `toml:"includeretryreason"`
+	DisableConsoleLogin            *bool               `toml:"disableconsolelogin"`
+}
+
+func (c *ConfigDTO) DriverConfig() (gosnowflake.Config, error) {
```

Comment:
nit: if we use the lib to read the toml files, it's most probably possible to provide custom unmarshaller for the few types needing additional logic here, so we could skip the additional dto + conversion logic. Can we add it to improvements in SNOW-933289